modules/local/star/align/main.nf

```diff
@@ -2,6 +2,7 @@ process STAR_ALIGN {
     tag "${meta.id}"
     label 'process_high'
 
+    conda "${moduleDir}/environment.yml"
```

Comment:
This change is pending in another elsewhere :) Could you please not include this in the final PR?